components/stacks-network/Cargo.toml

```diff
@@ -39,6 +39,8 @@ clarinet-files = { path = "../clarinet-files" }
 clarinet-deployments = { path = "../clarinet-deployments" }
 hiro-system-kit = { path = "../hiro-system-kit", features = ["log"] }
 stacks-codec = { path = "../stacks-codec" }
+flate2 = "1.1.1"
+tar = "0.4.44"
```

Comment:
We have this non-enforced convention in the repo where crates.io dependencies are listed first, followed by local dependencies.
Can you move these two new dependencies to lines 31 and 32?